client/my-sites/upgrades/domain-management/list/index.jsx

```diff
@@ -329,7 +330,11 @@ export const List = React.createClass( {
 			return times( 3, n => <ListItemPlaceholder key={ `item-${ n }` } /> );
 		}
 
-		return this.props.domains.list.map( ( domain, index ) => {
+		const domains = this.props.selectedSite.jetpack
```

Comment:
We should check first if `this.props.selectedSite` - we have enough of "`jetpack` is not defined" errors already 😀